Index: .gitlab-ci.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>stages:\r\n  - test\r\n  - build\r\n\r\njob_test:\r\n  stage: test\r\n  script:\r\n    - echo \"running tests.... \"\r\n    - npm test\r\n\r\njob_build:\r\n  stage: build\r\n  script:\r\n    - echo \"running build.... \"\r\n    - npm run build\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitlab-ci.yml b/.gitlab-ci.yml
--- a/.gitlab-ci.yml	(revision 3cae0c3c8e12b8afe7004c978dcca10453013cc5)
+++ b/.gitlab-ci.yml	(date 1749760693872)
@@ -7,9 +7,13 @@
   script:
     - echo "running tests.... "
     - npm test
+  tags:
+    - "groep19Runner"
 
 job_build:
   stage: build
   script:
     - echo "running build.... "
     - npm run build
+  tags:
+    - "groep19Runner"
\ No newline at end of file
